SUMMARY.md

```diff
@@ -119,10 +121,13 @@
 * [FAQ](faq/README.md)
   * [Security](faq/security.md)
   * [Security Self Assessment Questionnaire](faq/security-self-assessment.md)
-  * [Audit Logs](faq/audit.md)
-  * [Turn On Auditing](faq/turn-on-auditing.md)
-  * [Collect Microsoft 365 Data](faq/collect-office-365-data.md)
-  * [Migrate Audit Logs](faq/migrate-lucene-to-cosmos.md)
+  * [Activation](faq/activation-faq.md)
```

Comment:
u Summaryu posložiti redoslijed članaka kao što je u excelici. Activation faq article staviti poslije Licensing FAQ članka